docs/operator-manual/applicationset.yaml

```diff
@@ -178,14 +168,6 @@ spec:
     # Prevent an Application's child resources from being deleted, when the parent Application is deleted
     preserveResourcesOnDeletion: true
 
-    # which fields of the ApplicationSet should be ignored when comparing Applications.
-    ignoreApplicationDifferences:
```

Comment:
Validation error from this:
```
decoding error: unknown field "spec.syncPolicy.ignoreApplicationDifferences"
```
Note: `ignoreApplicationDifferences` already appears below where it actually belongs

Reply:
Good catch, after this is cleaned up I think we'll be ready to merge.

Reply:
I don't understand. It is cleaned up (exists where it should below and not here). WDYM?